### ISSUE TITLE
architectures

### ISSUE THREAD

**Issue by martindurant** [conda-forge/crc32c-feedstock#26]
### Comment:

Looking at https://anaconda.org/conda-forge/crc32c/files , I don't see mac-arm or linux-aarch/ppc builds. I see nothing in the recipe to prevent those - do you you know what's become of them?

**Comment by jakirkham** [comment]
Addressing this would involve adding this feedstock to respective arch migrators. Did this in PR: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6692

The bot should then generate PRs to add the different architectures here

Looks like the bot has already added one for macOS ARM: https://github.com/conda-forge/crc32c-feedstock/pull/37

**Comment by jhamman** [comment]
@jakirkham - it looks like the bot opened those arch PRs but they haven't been merged yet. Can we help move those along somehow? Zarr3 on mac users are now stuck behind this issue as well.

**Comment by jakirkham** [comment]
The maintainer already approved the macOS ARM PR. So went ahead and merged: https://github.com/conda-forge/crc32c-feedstock/pull/37

The Linux arch PR ( https://github.com/conda-forge/crc32c-feedstock/pull/38 ) currently is failing. So someone would need to debug and fix that

If we think this feedstock is important, we may want to find some candidates for maintainership here

**Comment by jhamman** [comment]
Thanks @jakirkham! 

Do we understand what is wrong with the Linux arch migration?

@pinin4fjords - you seem to be the only maintainer here (thank you!) Are you looking for additional folks to chip in. I suspect we can motivate folks from the Zarr project to chip in if so.

**Comment by jakirkham** [comment]
Unfortunately the logs can go stale after some time. Also that migrator may have conflicts with the macOS one that was already merged (as mentioned previously)

So asked the bot to create a new one, which it looks like it just did: https://github.com/conda-forge/crc32c-feedstock/pull/42

So would keep an eye on that to see what the logs reveal

**Comment by jakirkham** [comment]
The new bot PR passes. So it looks like it just needs a review. Asked the maintainer in that PR if they could take a look

**Comment by pinin4fjords** [comment]
> @pinin4fjords - you seem to be the only maintainer here (thank you!) Are you looking for additional folks to chip in. I suspect we can motivate folks from the Zarr project to chip in if so.

@jhamman Yes please! I don't spend enough time in conda-forge to understand what's going on any more. I should probably not be the maintainer.

**Comment by jakirkham** [comment]
There is now an issue soliciting additional maintainers: https://github.com/conda-forge/crc32c-feedstock/issues/43

Let's direct some folks who would be interested in chipping in there
